doc/kernel/usermode/mpu_stack_objects.rst

```diff
@@ -0,0 +1,49 @@
+.. _mpu_stack_objects:
+
+MPU Stack Objects
+#################
+
+Thread Stack Creation
+=====================
```

Comment:
our convention is to use asterisks for H2 headings  (equal signs are for H3 headings)

Reply:
ill change it to *